### ISSUE TITLE
if the user sets "Block sites from setting ant data" in preferences -> Cokies section on Opera 44, hola player doesn't load properly.

### ISSUE THREAD

**Issue by mahmutbayri** [hola/hola_player#6]
My test url is http://hola.github.io/examples/cdn/#hola_player_ima

The player try to read data from localStorage object.

Full message that given by console:

```
videojs5-hlsjs-source-handler.js:4Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at Object.d.88 (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:12:637)
    at e (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:560)
    at http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:611
    at Object.d.1.@hola.org/hls.js (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:823)
    at e (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:560)
    at http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:611
    at Object.d.247../css/dvr.css (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:25:11005)
    at e (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:560)
    at a (http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:726)
    at http://cdn.jsdelivr.net/hola_player/1.0.24/hola_player.js?customer=demo:1:743
d.88 @ videojs5-hlsjs-source-handler.js:4
e @ _prelude.js:1
(anonymous) @ _prelude.js:1
d.1.@hola.org/hls.js @ hola_videojs_hls.js:2
e @ _prelude.js:1
(anonymous) @ _prelude.js:1
d.247../css/dvr.css @ hola_player.js:7
e @ _prelude.js:1
a @ _prelude.js:1
(anonymous) @ _prelude.js:1
(anonymous) @ _prelude.js:1
(anonymous) @ _prelude.js:1
VM64:7 Uncaught TypeError: window.hola_player is not a function
    at VM64:7
```
 
Thanks.

**Comment by akashta** [comment]
fixed in 1.0.26

**Comment by mahmutbayri** [comment]
now it works, thanks.
